modules/oc-mirror-installing-plugin.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="installation-oc-mirror-installing-plugin_{context}"]
+= Installing the oc-mirror OpenShift CLI plug-in
+
+// TODO: intro
+
+// You can download and install the oc-mirror CLI plug-in to... TODO
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Download the oc-mirror CLI plug-in.
+
+.. Navigate to the link:https://console.redhat.com/openshift/downloads[Downloads] page of the {console-redhat-com}.
+
+.. Under the *Command-line interface (CLI) tools* section, click *Download* for *TODO*.
++
+// TODO: What will it be listed as?
+// TODO: Will they need to select OS/arch?
```

Comment:
@afflom @jpower432 Also, do we know if they'll have to select OS/arch? And will the file be called just `oc-mirror`, or will there be any extensions or folders?

Reply:
Thanks, we might just have to wait til we get closer to know this info. I'll add reminders to check up on these questions